pkg/ddc/alluxio/ufs.go

```
@@ -46,6 +46,26 @@ func (e *AlluxioEngine) ShouldCheckUFS() (should bool, err error) {
 	return
 }
 
+func (e *AlluxioEngine) GetUpdateUFSMap() (map[string][]string, error) {
+	updateUFSMap := make(map[string][]string)
+	// For Alluxio Engine, always attempt to prepare UFS
+	resultInCtx, resultHaveMounted, err := e.getMounts()
```

Better to check if err is nil to fail fast